api/python/quilt3/telemetry.py

```diff
@@ -0,0 +1,118 @@
+import functools
+from concurrent.futures import ThreadPoolExecutor, wait
+from requests_futures.sessions import FuturesSession
+from threading import Lock
+import uuid
+import atexit
+import time
+
+TELEMETRY_URL = "https://gha7zraiy7.execute-api.us-east-1.amazonaws.com/Stage/metrics/installer"
+TELEMETRY_USER_AGENT = "QuiltCli"
+API_CALL_TIMEOUT_SECS = 5
```

Comment:
If it's all async, then it probably doesn't matter what the timeout is?

Reply:
I think we can get rid of the timeout as long as we confirm that we can make a connection from the client to the API.